index.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+## search for all editors and plugins
+declare -a arr=(`find . -name "meta.yaml"`)
+FIRST_LINE=true
+echo "["
+## now loop through meta files
+for i in "${arr[@]}"
+do
+    IFS='/' segments=($i)
```

Comment:
Can you elaborate what this command does?

Reply:
splitting path variable on segments